libs/langchain/tests/integration_tests/embeddings/test_openai.py

```diff
@@ -71,6 +71,38 @@ def test_openai_embedding_with_empty_string() -> None:
     assert len(output[1]) == 1536
 
 
+@pytest.mark.asyncio
+async def test_sync_async_embed_documents_equal() -> None:
+    documents = ["foo bar", "bar foo", "foo"]
+    embedding = OpenAIEmbeddings()
+    sync_output = embedding.embed_documents(documents)
+    async_output = await embedding.aembed_documents(documents)
+    assert np.isclose(sync_output, async_output, atol=1e-4).sum() > 0.99
+
+
+@pytest.mark.asyncio
+async def test_sync_async_embed_query_equal() -> None:
+    query = "foo bar"
+    embedding = OpenAIEmbeddings()
+    sync_output = embedding.embed_query(query)
+    async_output = await embedding.aembed_query(query)
+    assert np.isclose(sync_output, async_output, atol=1e-4).sum() > 0.99
+
+
+@pytest.mark.asyncio
+async def test_with_without_tokenizing_equal() -> None:
+    documents = ["foo bar baz bum buzz boom"]
+    with_tokenizing = OpenAIEmbeddings()
+    without_tokenizing = OpenAIEmbeddings(embedding_ctx_length=None)
+    tokenize_output = with_tokenizing.embed_documents(documents)
+    text_output = without_tokenizing.embed_documents(documents)
+    assert np.isclose(tokenize_output, text_output, atol=1e-4).sum() > 0.99
```

Comment:
admit this isn't great, shouldn't be an integration test. but we'll need to do lots of cleanup before we run them in CI anyways, so doesn't matter too much for now imo